src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/ComponentBroadcaster.cs

```diff
@@ -133,93 +131,55 @@ public void OnFrameCompleted(SocketEndpointConnectionDelta connectionDelta)
                     {
                         EnsureComponentInitialized();
 
-                        endpointsNeedingCompleteChanges.Clear();
-                        endpointsNeedingDeltaChanges.Clear();
-
-                        filteredEndpointsNeedingCompleteChanges.Clear();
-                        filteredEndpointsNeedingDeltaChanges.Clear();
-                        filteredEndpointsNotNeedingDeltaChanges.Clear();
-
-                        foreach (SocketEndpoint endpoint in connectionDelta.AddedConnections)
+                        if (TransformBroadcaster != null)

```

Comment:
>if (TransformBroadcaster != null) [](start = 24, length = 33)

Have you seen a case where the TransformBroadcaster becomes null here? It's already checked for null up a few lines, so it would have to be OnFrameCompleted or BeginUpdatingFrame that destroys it

Reply:
I'll remove this, this was primarily a check to make sure a TransformBroadcaster referenced itself. I've never seen this fired.